package.json

```diff
@@ -67,6 +67,69 @@
   ],
   "main": "./dist/main",
   "contributes": {
+    "languages": [
```

Comment:
Why is this `languages` contribution point here? It shouldn't be, could you please remove?

Reply:
Other than this, I think everything looks good!

Reply:
Thanks! Yeah I have no idea where that came from, I certainly didn't add it manually. Anyways, I removed it in e640ae4cf61d21e5ce6e15d425050b91250db021 
If you'd like me to squash it into one commit to keep the commit log clean, I can do so as well if you want.